src/components/wasteCalendar/WasteReminderSettings.tsx

```diff
@@ -0,0 +1,361 @@
+import React, { useCallback, useEffect, useReducer, useState } from 'react';
+import {
+  Alert,
+  FlatList,
+  Modal,
+  RefreshControl,
+  SafeAreaView,
+  ScrollView,
+  StyleSheet,
+  TouchableOpacity,
+  View
+} from 'react-native';
+import DateTimePicker from '@react-native-community/datetimepicker';
+import { ListItem, normalize } from 'react-native-elements';
+
+import {
+  deleteReminderSetting,
+  getReminderSettings,
+  updateReminderSettings
+} from '../../pushNotifications';
+import { Switch } from '../Switch';
+import { BoldText, RegularText } from '../Text';
+import { Touchable } from '../Touchable';
+import { Radiobutton } from '../Radiobutton';
+import { Wrapper, WrapperHorizontal, WrapperRow } from '../Wrapper';
+import { colors, device, texts } from '../../config';
+import { ReminderSettings, WasteTypeData } from '../../types';
+import {
+  ReminderSettingsAction,
+  ReminderSettingsActionType,
+  reminderSettingsReducer
+} from './ReminderSettingsReducer';
+import { Button } from '../Button';
+import { areValidReminderSettings, parseReminderSettings } from '../../jsonValidation';
+import { SectionHeader } from '../SectionHeader';
+
+const showErrorAlert = () => {
+  Alert.alert(texts.wasteCalendar.errorOnUpdateTitle, texts.wasteCalendar.errorOnUpdateBody);
+};
+
+const initialReminderTime = new Date();
+initialReminderTime.setHours(9);
+initialReminderTime.setMinutes(0);
```

Comment:
did we know at what times the waste gets collected? can 9:00 be too late on the same day if somebody does not change the initial value so we should suggest an earlier time?

Reply:
we do not know the time, and i just defaulted to some value.
the user setting up the reminder will know when they need the reminder. i do not see the benefit of adding any specific time there